pyramid_oereb/lib/readers/extract.py

```diff
@@ -141,15 +141,17 @@ def read(self, real_estate, municipality, params):
 
         if municipality.published:
 
+            position = 1
             for plr_source in self._plr_sources_:
```

Comment:
`for position, plr_source in enumerate(self._plr_sources_, start=1):`

Reply:
Done